packages/showcases/vue/stories/components/structure/VtmnDivider/VtmnDivider.stories.js

```diff
@@ -16,7 +16,7 @@ const Template = (args) => ({
   setup() {
     return { args };
   },
-  template: `<div style="width: 400px; height: 250px;"><VtmnDivider v-bind="args">Label</VtmnDivider></div>`,
+  template: `<div style="width: 400px; height: 250px;"><VtmnDivider v-bind="args" aria-labelledby="title-test">Label</VtmnDivider></div>`,
```

Comment:
And where is the element with the id `#title-test`? If you add this, you need to add this element in order to label the divider.